scripts/init-green-db.sh

```
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+# Initialize green database
+echo "Initializing green database..."
+
+# Create database if it doesn't exist
+docker exec postgres-green psql -U spheroseg -c "CREATE DATABASE spheroseg_green;" 2>/dev/null || echo "Database already exists"
+
+# Run Prisma migrations
+cd /home/cvat/cell-segmentation-hub/backend
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)